### PR TITLE
First approach to positive-number validator

### PR DESCRIPTION
**Pull request by v-borrego**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,14 +29,29 @@ const validationSchema = {
 };
 ```
 
+We can specify if the validator allows zero (true by default):
+
+```javascript
+import { minNumber } from '@lemoncode/fonk-min-number-validator';
```
Comment (nasdan) [inline README.md]: fonk-positive-number-validator

File: src/validator.ts [hunk 2]
```diff
@@ -1,27 +1,45 @@
...
-  const succeeded = !isDefined(value) || ...;
+  const succeeded =
+    !isDefined(value) || (validateType(value) && validate(value, args));
 
   return {
     succeeded,
-    message: succeeded
-      ? ''
-      : buildCustomMessage((message as string) || defaultMessage, args),
+    message: succeeded ? '' : (message as string),
```
Comment (nasdan) [inline src/validator.ts]: I would left the `parseMessageWithCustomArgs` for all validators with customArgs, even in this case where it's apparently not necessary.

File: README.md (hunk 1 above)
Comment (v-borrego) [reply README.md]: Done

File: src/validator.ts (hunk 2 above)
Comment (v-borrego) [reply src/validator.ts]: Done

**Pull request merged**